org.osgi.test.junit5.cm/test.bndrun

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) OSGi Alliance (2019, 2020). All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+-tester: biz.aQute.tester.junit-platform
+
+-runvm: ${def;argLine}
+
+-runfw: org.eclipse.osgi
+-resolve.effective: active
+-runproperties: \
+	logback.configurationFile=file:${.}/logback.xml,\
+	org.apache.felix.http.host=localhost,\
+	org.osgi.service.http.port=*,\
+	org.osgi.framework.bootdelegation=sun.reflect,\
+	osgi.console=
+	
+-runsystempackages: \
+	org.slf4j;version=1.7.25,\
+	org.slf4j.helpers;version=1.7.25,\
+	org.slf4j.spi;version=1.7.25
+-runpath: \
+	ch.qos.logback.classic,\
+	ch.qos.logback.core,\
+	org.apache.felix.logback,\
+	slf4j.api
+-runrequires: \
+	bnd.identity;id='${project.artifactId}-tests',\
+	bnd.identity;id='junit-jupiter-engine',\
+	bnd.identity;id='junit-platform-launcher'
+# Please try and keep this sorted to make diffing more consistent
+-runbundles: \
+	junit-jupiter-api;version='[5.6.2,5.6.3)',\
+	junit-jupiter-engine;version='[5.6.2,5.6.3)',\
+	junit-platform-commons;version='[1.6.2,1.6.3)',\
+	junit-platform-engine;version='[1.6.2,1.6.3)',\
+	junit-platform-launcher;version='[1.6.2,1.6.3)',\
+	org.opentest4j;version='[1.2.0,1.2.1)',\
+	org.osgi.test.common;version='[0.10.0,0.10.1)',\
+	org.osgi.test.junit5.cm-tests;version='[0.10.0,0.10.1)',\
+	org.osgi.service.cm;version='[1.6.0,1.6.1)',\
+	assertj-core;version='[3.16.1,3.16.2)',\
+	org.apache.felix.configadmin;version='[1.9.16,1.9.17)',\
+	org.osgi.test.junit5.cm;version='[0.10.0,0.10.1)'
+-runee: JavaSE-11
```

Comment:
We shouldn't set this - Maven sets it based on the Maven environment (or if you must set it for some reason, it should probably be set to JavaSE-1.8). This is what the other integration tests due.